### PR TITLE
Clarify accessor.count for morph targets

### LINKED ISSUE #1339: Morph Targets with different accessors count
What are the rules in place if I have a

```
"meshes": [
    {
      "primitives": [
        {
          ...
          "targets": [
            {
              "NORMAL": 3,
              "POSITION": 4,
              "TANGENT": 5
            }
        ...
```

And the `accessors` at 3, 4, and 5 have different `accessors.count`

First I am not sure if a model can have a different number of `NORMAL` vec3 then `POSITION` since its all a per-vertex change.

I guess not sure if this something that is implict and not stated in the spec or something to be aware of when parsing to not assume the same count, which if later then how does one handle that

### PR DESCRIPTION
**Pull request by lexaknyazev**
Fixes #1339.

### CODE REVIEW

File: specification/2.0/README.md [hunk 1]
```diff
@@ -792,6 +792,8 @@ Valid accessor type and component type for each attribute semantic property are
 
 `POSITION` accessor **must** have `min` and `max` properties defined.
 
+All Morph Target's accessors **must** have the same `count` which must match the `count` of accessors of the original mesh.
```
Comment (emackey) [inline specification/2.0/README.md]: You use `must` twice for the same directive.  How about something like this?

> The `count` of each Morph Target accessor **must** exactly match the `count` of the accessor for the original mesh.

Comment (lexaknyazev) [reply specification/2.0/README.md]: > The `count` of each Morph Target accessor **must** exactly match the `count` of the accessor for the original mesh.

Is it clear what "the accessor" means?
We're trying to say that all vertex data accessors have the same `count` - no matter original mesh or morph target.

Comment (emackey) [reply specification/2.0/README.md]: How about:

> All Morph Target's accessors **must** have the same `count` as the accessors of the original mesh.

or

> For a given mesh, all vertex data accessors, including morph target accessors, **must** have the same `count`.

Comment (emackey) [reply specification/2.0/README.md]: or replace "mesh" with "primitive"

**Pull request merged**